releasenotes/notes/0.17/fix-piecewise-breakpoints-21db5a7823fdfbed.yaml

```diff
@@ -0,0 +1,10 @@
+---
+fixes:
+  - |
+    Fixed the handling of breakpoints in the
+    :class:`~qiskit.circuit.library.PiecewisePolynomialPauliRotations` class
+    in the :mod:`qiskit.circuit.library`. Now for ``n`` intervals,
+    ``n+1`` breakpoints are allowed. This enables specifying another end
+    interval other than :math:`2^{num qubits}`. This is important because
+    from the end of the last interval to `:math:`2^{num qubits}` the function
```

Comment:
```suggestion
    interval other than :math:`2^\text{num qubits}`. This is important because
    from the end of the last interval to :math:`2^\text{num qubits}` the function
```